sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -415,44 +474,9 @@ describes available configurations for retries, logging, transport protocols, an
-* Recognize receipts from a URL: [sample_recognize_receipts_from_url.py][sample_recognize_receipts_from_url]
-* Recognize business cards: [sample_recognize_business_cards.py][sample_recognize_business_cards]
-* Recognize invoices: [sample_recognize_invoices.py][sample_recognize_invoices]
-* Recognize identity documents: [sample_recognize_identity_documents.py][sample_recognize_identity_documents]
-* Recognize content: [sample_recognize_content.py][sample_recognize_content]
-* Recognize custom forms: [sample_recognize_custom_forms.py][sample_recognize_custom_forms]
-* Train a model without labels: [sample_train_model_without_labels.py][sample_train_model_without_labels]
-* Train a model with labels: [sample_train_model_with_labels.py][sample_train_model_with_labels]
-* Manage custom models: [sample_manage_custom_models.py][sample_manage_custom_models]
-* Copy a model between Form Recognizer resources: [sample_copy_model.py][sample_copy_model]
-* Create a composed model from a collection of models trained with labels: [sample_create_composed_model.py][sample_create_composed_model]
-
-### Async APIs
-This library also includes a complete async API supported on Python 3.6+. To use it, you must
-first install an async transport, such as [aiohttp](https://pypi.org/project/aiohttp/). Async clients
-are found under the `azure.ai.formrecognizer.aio` namespace.
-
-* Client authentication: [sample_authentication_async.py][sample_authentication_async]
-* Recognize receipts: [sample_recognize_receipts_async.py][sample_recognize_receipts_async]
-* Recognize receipts from a URL: [sample_recognize_receipts_from_url_async.py][sample_recognize_receipts_from_url_async]
-* Recognize business cards: [sample_recognize_business_cards_async.py][sample_recognize_business_cards_async]
-* Recognize invoices: [sample_recognize_invoices_async.py][sample_recognize_invoices_async]
-* Recognize identity documents: [sample_recognize_identity_documents_async.py][sample_recognize_identity_documents_async]
-* Recognize content: [sample_recognize_content_async.py][sample_recognize_content_async]
-* Recognize custom forms: [sample_recognize_custom_forms_async.py][sample_recognize_custom_forms_async]
-* Train a model without labels: [sample_train_model_without_labels_async.py][sample_train_model_without_labels_async]
-* Train a model with labels: [sample_train_model_with_labels_async.py][sample_train_model_with_labels_async]
-* Manage custom models: [sample_manage_custom_models_async.py][sample_manage_custom_models_async]
-* Copy a model between Form Recognizer resources: [sample_copy_model_async.py][sample_copy_model_async]
-* Create a composed model from a collection of models trained with labels: [sample_create_composed_model_async.py][sample_create_composed_model_async]
+See the [Sample README][sample_readme] for several code snippets illustrating common patterns used in the Form Recognizer Python API.
```

Comment:
sample readme will be updated in a future PR, right?

Reply:
yep! https://github.com/Azure/azure-sdk-for-python/issues/20841